src/libs/ReportUtils.js

```diff
@@ -1707,6 +1707,39 @@ function canLeaveRoom(report, isPolicyMember) {
     return true;
 }
 
+/**
+ * Returns display names for those that can see the whisper.
+ * However, it returns "you" if only the current user can see it besides the person that sent it.
+ *
+ * @param {string[]} participants
+ * @returns {string}
+ */
+function getWhisperDisplayNames(participants) {
+    const isOneOnOneWhisper = this.isOnlyVisibleByCurrentUser(participants);
+
+    // If we removed all users, it means it's only visible to "you"
+    if (isOneOnOneWhisper) {
+        return Localize.translateLocal('common.youAfterPreposition');
+    }
+
+    const displayNames = [];
+    for (let i = 0; i < participants.length; i++) {
+        const login = participants[i];
+        displayNames.push(getDisplayNameForParticipant(login, !isOneOnOneWhisper));
+    }
+    return displayNames.join(', ');
+}
+
+/**
+ * Returns whether a whisper is only visible by the current user and whoever sent it.
+ * @param {string[]} participants
+ * @returns {Boolean}
+ */
+function isOnlyVisibleByCurrentUser(participants) {
+    const participantsWithoutCurrentUser = _.without(participants, sessionEmail);
+    return participantsWithoutCurrentUser.length === 0;
+}
```

Comment:
The logic here doesn't have anything to do with visibility or whispers. It seems like all it is doing is checking that the participants only contains the current user.

```suggestion
/**
 * @param {string[]} participants
 * @returns {Boolean}
 */
function isCurrentUserTheOnlyParticipant(participants) {
    return participants && participants.length === 1 && participants[0] === sessionEmail;
}
```